src/components/classifier/ClassifierButton.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import {
+    StyleSheet,
+    TouchableOpacity
+} from 'react-native'
+import FontedText from '../common/FontedText'
+import PropTypes from 'prop-types'
+import EStyleSheet from 'react-native-extended-stylesheet'
+
+const ClassifierButton = (props) => {
+    const buttonStyle = [props.style, styles.button]
+    const textStyle = []
+    switch (props.type) {
```

Comment:
Will there be a default here, or are these the only types you are expecting that will expand in the future?

Reply:
This could expand in the future. I prefer to leave out the default so that if this component is being used incorrectly it will brake the code